src/coreComponents/physicsSolvers/contact/ContactSolverBase.hpp

```diff
@@ -53,19 +52,19 @@ class ContactSolverBase : public SolverBase
 
   string const & getContactRelationName() const { return m_contactRelationName; }
 
-  string const & getFractureRegionName() const { return m_fractureRegionName; }
+  string const & getFractureRegionName() const { return m_fractureRegionNames[0]; }
```

Comment:
TODO double check

Reply:
yeah for now, there a few places where we basically only handle a single fracture Region. I don't think it's a big deal. But maybe we can change this function to: `getUniqueFractureRegionName()`

Reply:
how about throwing an error (for now) when there is more than one fracture region?

Reply:
added an error here https://github.com/GEOS-DEV/GEOS/blob/4fd813d740c65a17257e280ff669fea8782598e5/src/coreComponents/physicsSolvers/contact/ContactSolverBase.cpp#L113
hope it is temporary ok, until we further clean up the fracture region logic